src/test/java/io/phasetwo/service/importexport/OrganizationImportTest.java

```diff
@@ -0,0 +1,290 @@
+package io.phasetwo.service.importexport;
+
+import static io.phasetwo.service.Helpers.loadJson;
+import static io.phasetwo.service.Helpers.objectMapper;
+import static io.phasetwo.service.Orgs.ORG_OWNER_CONFIG_KEY;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import io.phasetwo.client.openapi.model.OrganizationRepresentation;
+import io.phasetwo.service.AbstractOrganizationTest;
+import io.phasetwo.service.importexport.representation.KeycloakOrgsRepresentation;
+import io.phasetwo.service.importexport.representation.OrganizationRoleRepresentation;
+import io.phasetwo.service.representation.OrganizationRole;
+import jakarta.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.List;
+import lombok.extern.jbosslog.JBossLog;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.keycloak.representations.idm.RealmRepresentation;
+
+@JBossLog
+public class OrganizationImportTest extends AbstractOrganizationTest {
```

Comment:
We should have a test for running an import that contains an organization that already exists.

Reply:
We have one in **OrganizationTransactionalImportTest**. Please check testOrganizationsTransactionalImport